content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,263 @@
+==========================
+Dispatch management system
+==========================
+
+The *dispatch management system* feature in Odoo is used to plan and build shipments. Key features
+include:
+
+- **Load building**: Group products for specific carriers, assign those :doc:`batches
+  <../picking_methods/batch>` to loading docks, and manage vehicle assignments based on fleet
+  capacity. This ensures the right products are packed into the appropriate trucks for delivery.
+- :doc:`Fleet management <../../../../hr/fleet>`: Track and manage the capacity of in-house
+  delivery vehicles.
+
+Configuration
+=============
+
+To use the dispatch management system, the following setup must be completed:
+
+#. Install the **Fleet** application.
+#. Configure vehicle :ref:`capacity (volume and weight)
+   <inventory/shipping_receiving/vehicle-capacity>`.
+#. Enter vehicle :doc:`car model(s) <../../../../hr/fleet/models>`.
+#. Enable :ref:`necessary features <inventory/shipping_receiving/inventory-features>` in the
+   **Inventory** app.
+#. Set up :ref:`vehicles as delivery methods
+   <inventory/shipping_receiving/delivery-method-for-carrier>`.
+#. Create :ref:`dock locations <inventory/shipping_receiving/docks>`.
+
+.. _inventory/shipping_receiving/vehicle-capacity:
+
+Vehicle capacity
+----------------
+
+To configure the vehicle capacity, go to :menuselection:`Fleet app --> Configuration -->
+Categories`.
+
+Next, click :guilabel:`New` to add a new category, or click into an existing category to modify it.
+In the :guilabel:`Name` field, enter the type of vehicle (e.g., `Pick-up truck`, `Van`, or `Cargo
+truck`). Then, enter the vehicle capacity in the :guilabel:`Max Weight` (in kilograms) and
+:guilabel:`Volume` (in cubic meters) fields. `
+
+.. note::
+   The units of measurement for vehicle capacity are assigned at the global level in the
+   **Settings** app, in the :guilabel:`Units of Measure` section.
+
+   .. image:: dispatch/settings.png
+      :alt: Show units of measure settings.
+
+.. seealso::
+   :ref:`Vehicle category <fleet/categories>`
+
+.. image:: dispatch/categories.png
+   :alt: Vehicle categories with defined weight and volume.
```

Comment:
I suggest swapping the order of these elements, so that the relevant image follows the paragraph, then the note admonition, then the seealso. Otherwise, the seealso seems to break the flow and image loses some context IMO
```suggestion

.. image:: dispatch/categories.png
   :alt: Vehicle categories with defined weight and volume.

.. note::
   The units of measurement for vehicle capacity are assigned at the global level in the
   **Settings** app, in the :guilabel:`Units of Measure` section.

   .. image:: dispatch/settings.png
      :alt: Show units of measure settings.

.. seealso::
   :ref:`Vehicle category <fleet/categories>`

```